test/functional/feature_backwards_compatibility.py

```diff
@@ -123,52 +124,52 @@ def run_test(self):
         wallet = node_v18.get_wallet_rpc("w1_v18")
         info = wallet.getwalletinfo()
         assert info['private_keys_enabled']
-        assert info['keypoolsize'] > 0
+        assert_greater_than(info['keypoolsize'] > 0)
```

Comment:
error here, `,` instead of `>`

Reply:
Ops, thanks to catch my error @meshcollider!